doc/nrf/gs_installing.rst

```diff
@@ -11,10 +10,586 @@ See the :ref:`gs_assistant` section for information about how to install the Get
+               :alt: nRF Connect SDK options in SES on Windows
+
+               nRF Connect SDK options in SES (Windows)
+
+         Use this section to change the SES environment settings later as well.
+
+      #. Specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable (for example, ``/usr/local/bin/west``):
+
+          .. parsed-literal::
+             :class: highlight
+
+             -DWEST=\ *path_to_west*
+
+
+      If you start SES from the command line, it uses the global PATH variable to find the executables.
+      You do not need to explicitly configure the executables in SES.
+
+      Regardless of how you start SES, if you get an error that a tool or command cannot be found, first make sure that the tool is installed.
+      If it is installed, verify that its path is configured correctly in the SES settings or in the PATH variable.
+
+
+Changing the SES environment settings
+=====================================
+
+If you want to change the SES environment settings, click :guilabel:`Tools` -> :guilabel:`Options` and select the :guilabel:`nRF Connect` tab, as shown on the following screenshot from the Windows installation.
+
+.. _ses_options_figure:
+
+.. figure:: images/ses_options.png
+     :alt: nRF Connect SDK options in SES on Windows
+
+     nRF Connect SDK options in SES (Windows)
+
+If you want to configure tools that are not listed in the SES options, add them to the PATH variable.
+
+.. _build_environment_cli_win:
+
+Setting up the command line build environment
+=============================================
+
+If you want to build and program your application from the command line, you must set up your build environment by defining the required environment variables every time you open a new |bash_win| or |bash_linux_mac|.
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      To define the environment variables, navigate to the ``ncs`` folder and enter the following command: |envfile_win|
+
+      If you need to define additional environment variables, create the file |rcfile_win| and add the variables there.
+      This file is loaded automatically when you run the above command.
+
+   .. group-tab:: Linux
+
+      To define the environment variables, navigate to the ``ncs`` folder and enter the following command: |envfile_linux_mac|
+
+      If you need to define additional environment variables, create the file |rcfile_linux_mac| and add the variables there.
+      This file is loaded automatically when you run the above command.
+
+   .. group-tab:: macOS
+
+      To define the environment variables, navigate to the ``ncs`` folder and enter the following command: |envfile_linux_mac|
+
+      If you need to define additional environment variables, create the file |rcfile_linux_mac| and add the variables there.
+      This file is loaded automatically when you run the above command.
+
 
 .. toctree::
 
    gs_ins_os
```

Comment:
We can remove this if we are referring to this file at the begining of the document.
````suggestion
````

Reply:
Will move this to getting_started.rst. Otherwise, if deleted, this will remove the page from the hierarchy.